Cargo.toml

```diff
@@ -28,16 +28,16 @@ failure = "0.1.1"
 filetime = "0.1"
 flate2 = "1.0"
 fs2 = "0.4"
-git2 = "0.6.11"
-git2-curl = "0.7"
+git2 = "0.7.0"
```

Comment:
Don't know if this is intentional or not, but looks like https://github.com/alexcrichton/git2-rs does not contain commit which actually bumps the versions?

Reply:
Oops forgot to do `git push`